### PR TITLE
Added IsEvenOrOdd method

### LINKED ISSUE #9: Write a method to determine if a number is even or odd
Input: get a number from the user
Calculate if it is even or odd

### PR DESCRIPTION
**Pull request by drtaupier**
image

### CODE REVIEW

**Comment by mriverap** [comment]
Please change the screenshot to a sample of the execution.

File: CSharpBasics/BasicProblems.cs [hunk 1]
```diff
@@ -77,6 +79,15 @@ public static void IsEvenOrOdd()
             Console.Write("Please, enter a number: ");
             var inputNumber = Console.ReadLine();
             int.TryParse(inputNumber, out int num1);
+
+            if (num1 % 2 == 0)
```
Comment (mriverap) [inline CSharpBasics/BasicProblems.cs:83]: Great!

**Review by mriverap — CHANGES_REQUESTED** [review]
Link the appropriate Issue to this PR.

File: CSharpBasics/BasicProblems.cs (hunk 1 above)
Comment (drtaupier) [reply CSharpBasics/BasicProblems.cs:83]: image

Comment (drtaupier) [reply CSharpBasics/BasicProblems.cs:83]: image

**Review by mriverap — APPROVED** [review]

**Pull request merged**